### ISSUE TITLE
Re-add FreeBSD build

### ISSUE THREAD

**Issue by einsibjarni** [grafana/grafana-zabbix#1786]
**Is your feature request related to a problem? Please describe.**
Builds for FreeBSD where added in https://github.com/grafana/grafana-zabbix/pull/1501
Since 4.4.1 they've been missing.

**Describe the solution you'd like**
I really liked having signed binary builds. Hopefully, when the build process was updated, FreeBSD just got forgotten and it's an easy fix.

**Describe alternatives you've considered**

**Additional context**

**Comment by einsibjarni** [comment]
Yeah, I tried building it myself, and failed miserably. After trying before to build the grafana image renderer on freebsd, this was the last straw and I've migrated my grafana install to Debian :(

**Comment by dbaio** [comment]
> Yeah, I tried building it myself, and failed miserably. After trying before to build the grafana image renderer on freebsd, this was the last straw and I've migrated my grafana install to Debian :(

Hi,

I managed to build this plugin on a FreeBSD box and got it running with Grafana v11.4.0. Here is what I did, in case it helps you or anyone else.

```
# pkg install git-lite go mage npm yarn
# gmake install
# gmake dist
```


The only caveat is that when I started Grafana on my ARM64 host, it complained that the datasource/gpx_zabbix-datasource_linux_arm64 was missing, so I also had to do this (after `gmake dist`):

```
# cp -p dist/gpx_zabbix-plugin_freebsd_amd64 dist/datasource/gpx_zabbix-datasource_freebsd_amd64
# cp -p dist/gpx_zabbix-plugin_freebsd_arm64 dist/datasource/gpx_zabbix-datasource_freebsd_arm64
```

This is the plugin file list (after the above copy), for reference only:

```
root@grafana.xxx:/var/db/grafana/plugins/alexanderzobnin-zabbix-app # find .
.
./styles
./styles/light.css
./styles/dark.css
./module.js.map
./gpx_zabbix-plugin_freebsd_amd64
./LICENSE
./panel-triggers
./panel-triggers/module.js
./panel-triggers/img
./panel-triggers/img/icn-zabbix-problems-panel.svg
./panel-triggers/plugin.json
./panel-triggers/module.js.map
./panel-triggers/module.js.LICENSE.txt
./datasource
./datasource/plugin.json
./datasource/module.js
./datasource/gpx_zabbix-datasource_freebsd_arm64
./datasource/gpx_zabbix-datasource_linux_arm64
./datasource/img
./datasource/img/icn-zabbix-datasource.svg
./datasource/module.js.LICENSE.txt
./datasource/gpx_zabbix-datasource_windows_amd64.exe
./datasource/gpx_zabbix-datasource_darwin_amd64
./datasource/gpx_zabbix-datasource_linux_arm
./datasource/module.js.map
./datasource/dashboards
./datasource/dashboards/template_linux_server.json
./datasource/dashboards/zabbix_system_status.json
./datasource/dashboards/zabbix_server_dashboard.json
./datasource/gpx_zabbix-datasource_freebsd_amd64
./datasource/gpx_zabbix-datasource_darwin_arm64
./datasource/gpx_zabbix-datasource_linux_amd64
./datasource/query_help.md
./CHANGELOG.md
./go_plugin_build_manifest
./plugin.json
./module.js
./gpx_zabbix-plugin_freebsd_arm64
./README.md
./img
./img/screenshot-showcase.png
./img/icn-zabbix-app.svg
./img/screenshot-annotations.png
./img/screenshot-dashboard01.png
./img/screenshot-metric_editor.png
./img/screenshot-triggers.png
```

**Comment by dbaio** [comment]
@yesoreyeram Hi, If it's interesting, we can host a FreeBSD Cirrus-CI worker in the FreeBSD cluster infrastructure. Let me know, and I can arrange that.

**Comment by Alexander237** [comment]
It would be great to recover FreeBSD build.
Today I tested zabbix plugin v6.3.2 with grafana 12.4.2 on FreeBSD and it works 
cd /usr/local/src
git clone https://github.com/grafana/grafana-zabbix.git
cd grafana-zabbix
git checkout v6.3.2

cd cd /usr/local/src/grafana-zabbox

GOOS=freebsd GOARCH=amd64 go build \
  -o gpx_zabbix-datasource_freebsd_amd64 \
  ./pkg

cp gpx_zabbix-datasource_freebsd_amd64 /var/db/grafana/plugins/alexanderzobnin-zabbix-app/datasource/

chmod 755 /var/db/grafana/plugins/alexanderzobnin-zabbix-app/datasource/gpx_zabbix-datasource_freebsd_amd64
chown grafana:grafana /var/db/grafana/plugins/alexanderzobnin-zabbix-app/datasource/gpx_zabbix-datasource_freebsd_amd64

Grafana.ini
[plugins]
allow_loading_unsigned_plugins = alexanderzobnin-zabbix-app,alexanderzobnin-zabbix-datasource,alexanderzobnin-zabbix-triggers-panel

If I remove MANIFEST.txt from original zabbix plugin and restart grafana, then plugin works perfectly.
